examples/multiple-key-fields/multiple_key_fields.md

```diff
@@ -1,6 +1,10 @@
 # Multiple key fields
 
-A `table` provides access to its members using a key that comes from the `read-only` fields of the member. It is a key sequence, which is used to provide keyed access to its members. The key sequence is an ordered sequence of field names.
+A `table` provides access to its members using a key that comes from the `readonly` fields of the member. It is a key sequence, which is used to provide keyed access to its members. The key sequence is an ordered sequence of the field names.
+
+## Related links
+- [Table Syntax](/learn/by-example/table-syntax/)
+- [Structured keys](/learn/by-example/multiple-key-fields/)
```

Comment:
May be we can add `Maps` BBE as a related link as the table rows are mappings :thinking: